docs/devcontainers/faq.md

```diff
@@ -0,0 +1,85 @@
+---
+Order: 16
+Area: devcontainers
+TOCTitle: FAQ
+PageTitle: Visual Studio Code Dev Containers Frequently Asked Questions
+ContentId: c4784db6-ab00-4ac7-bca8-88edb638c593
+MetaDescription: Visual Studio Code Remote Development troubleshooting tips and tricks for Dev Containers
+DateApproved: 10/6/2022
+---
+# Dev Containers FAQ
+
+This article includes some of the common questions for getting the Dev Containers extension up and running in different environments.
+
+## Do "dev container definitions" define how an application is deployed?
+
+No. A development container defines an environment in which you develop your application before you are ready to deploy. While deployment and development containers may resemble one another, you may not want to include tools in a deployment image that you use during development.
+
+The [vscode-dev-containers repository](https://aka.ms/vscode-dev-containers) includes a set of dev container definitions for some common development environments. You can also [attach to a running container](/docs/devcontainers/attach-container.md) without setting up a dev container definition, if you prefer to use an alternate container build or deployment workflow.
```

Comment:
There will be a variety of spots we need to update from vscode-dev-containers to instead point to the new repos in the devcontainers org as part of the spec. I'll save most of that for future PRs, but since this stood out to me, maybe we at least update this one:

```suggestion
The [devcontainers/templates repository](https://github.com/devcontainers/templates) includes a set of dev container definitions for some common development environments. You can also [attach to a running container](/docs/devcontainers/attach-container.md) without setting up a dev container definition, if you prefer to use an alternate container build or deployment workflow.
```